server/src/main/scala/com/cloudera/livy/server/interactive/InteractiveSession.scala

```diff
@@ -202,6 +204,21 @@ class InteractiveSession(
     }
   }
 
+  private def findSparkRArchives(): Seq[String] = {
+    sys.env.get("SPARKR_ARCHIVES_PATH")
+      .map(_.split(",").toSeq)
+      .getOrElse {
+        sys.env.get("SPARK_HOME").map { case sparkHome =>
+          val rLibPath = Seq(sparkHome, "R", "lib").mkString(File.separator)
+          val rArchivesFile = new File(rLibPath, "sparkr.zip")
```

Comment:
yes. i agree. 
It needs to be added in a future README file (Prerequisites section)